docs/migrations/v6-to-v7.md

```diff
@@ -173,3 +173,29 @@ Keeper function `CheckMisbehaviourAndUpdateState` has been removed since functio
 `MsgSubmitMisbehaviour` is deprecated since `MsgUpdateClient` can now submit a `ClientMessage` type which can be any `Misbehaviour` implementations.
 
 The field `header` in `MsgUpdateClient` has been renamed to `client_message`.
+
+
+### IBC module constants
+
+IBC module constants have been moved from the `host` package to the `exported` package. Any usages will need to be updated.
+
+```diff
+import (
+    // ...
+-   host "github.com/cosmos/ibc-go/v6/modules/core/24-host"
++   ibcexported "github.com/cosmos/ibc-go/v6/modules/core/exported"
+    // ...
+)
+
+- host.ModuleName
++ ibcexported.ModuleName
+
+- host.StoreKey
++ ibcexported.StoreKey
+
+- host.QuerierRoute
++ ibcexported.QuerierRoute
+
+- host.RouterKey
++ ibcexported.RouterKey
+```
```

Comment:
I've added an entry to handle migrations, this one is a little different from usual, but let me know if you think this is clear

cc @crodriguezvega  / @damiannolan